iocage_lib/ioc_start.py

```diff
@@ -367,6 +367,9 @@ def __start_jail__(self):
             x for x in net
             + [x for x in parameters if '1' in x]
             + [
+                _sysvmsg,
+                _sysvsem,
+                _sysvshm,
                 f'name=ioc-{self.uuid}',
```

Comment:
```suggestion
                 f'name=ioc-{self.uuid}',
                 _sysvmsg,
                 _sysvsem,
                 _sysvshm,
```
Something similar to that. Not sure why Github is showing that topline change, may be the newline.